tests/model/PHPElementWithPHPDoc.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Model;
+
+abstract class PHPElementWithPHPDoc extends BasePHPElement
```

Comment:
Maybe make this one a trait? I have no strong opinion about this one though so up to you.

Reply:
May be..will think about it and look what looks better